src/components/KEmptyState/KEmptyState.vue

```diff
@@ -1,143 +1,173 @@
 <template>
   <section
```

Comment:
Hey @portikM 👋 

Glad you are doing this component now, I'm hoping to make use of its refactor shortly over in kuma, kinda been waiting for it to happen 🎉 

---

I noticed this is still using `section`. I 'think' we removed sections from KCard, should we replace this one here with a div also?

> The <section> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it. Sections should always have a heading, with very few exceptions.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

I know of a few areas in Kuma where our empty states aren't strictly a 'document section' and/or they will already appear in a `section` we've added in our application (i.e. `<section><KEmptyState /></section>`) so not having a section in the component would avoid a doubled up nested `section` i.e. `<section><section class="k-empty-state">...</section></section>`

lmk, not a huge request, but considering I think we removed sections in KCard this might have just been an oversight not to replace this with a `div`

Reply:
Agree, let’s switch to a div

Reply:
Done